draft-savage-ppm-3phm-mpc.md

```diff
@@ -223,11 +223,11 @@ are illustrated in {{fig-send-direction}}.
 ~~~ aasvg
 send                               send
 left:       .----.                 right:      .----.
-       .-->|  P₁  |---.                   .---|  P₁  |<--.
+       .---|  P₁  |<--.                   .-->|  P₁  |---.
       /     `----'     \                 /     `----'     \
-     /                  v               v                  \
+     v                  \               /                  v
  .----.                .----.       .----.                .----.
-|  P₂  |<-------------|  P₃  |     |  P₂  |------------->|  P₃  |
+|  P₃  |------------->|  P₂  |     |  P₃  |<-------------|  P₂  |
```

Comment:
Unfortunately, from the perspective of each helper in this picture, the directions no longer match the names.

Looking at these arrows, in the figure on the left (which is labeled as "send left"), P_3 is clearly sending a value to it's "right".

Reply:
Yes, but the other way around, P_1 and P_2 send values in the "wrong" direction. With a circle, you'll always have some going the "wrong" way. 

This change makes "send left" align with counter-clockwise and "send right" align with clockwise, and has the advantage that P_1 at the top is sending in the "correct" direction.

Reply:
Oh, I think I see what you're saying now. If you picture this as 3 people looking inward, then the original makes sense. That's not at all how I visualized it; I see it more like a clock, and it feels way more natural to think of left as counter clockwise and right as clockwise (righty-tighty, lefty-loosey...)

Here's an alternative:
```
~~~ aasvg
send
left:      .----.      .----.      .----.
        .-+  P₁  |<---+  P₂  |<---+  P₃  |<--.
         \ `----'      `----'      `----'   /
          '--------------------------------'

send
right:     .----.      .----.      .----.
      .-->|  P₁  +--->+  P₂  +--->|  P₃  +-.
       \   `----'      `----'      `----' /
        '--------------------------------'
~~~
```
which renders as:
<img width="429" alt="image" src="https://github.com/private-attribution/i-d/assets/982499/1b240641-ae9c-4c6f-b97a-ca81137e3000">


The down side to this approach is that making the top diagram consistent is a clearly not as nice looking:
<img width="395" alt="image" src="https://github.com/private-attribution/i-d/assets/982499/13148640-425d-46ca-a222-2879296ec9de">